### PR TITLE
Marcyfeature

### PR DESCRIPTION
**Pull request by gitgregor**
Ok Marcy I want merge it

### CODE REVIEW

File: file1.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
 Wstęp
 
-Ta książka poświęcona jest  współczesnym problemom ekonomicznym świata 
+Ta książka poświęcona jest  współczesnym problemom ekonomicznym świata.
```
Comment (gitgregor) [inline file1.txt:3]: Marcy , sentence with dot at the end of sentence is ok for me.

**Review by gitgregor — COMMENTED** [review]
looks good

File: file1.txt (hunk 1 above)
Comment (Marcytest) [reply file1.txt:3]: I've merged with master on remote by CLI in my local rep but nothing is changed. Do you know Why

Comment (gitgregor) [reply file1.txt:3]: Ok I am going to merge and let's see

**Pull request merged**